packages/react-form-renderer/README.md

```diff
@@ -26,15 +26,11 @@ Data Driven Forms is a React library used for rendering and managing forms with
 
 :book: For more information please visit the [documentation](https://data-driven-forms.org/). :book:
 
-Used by [ManageIQ](http://manageiq.org/), Red Hat Cloud Services.
 
 **Table of Contents**
 
 - [Installation](#installation)
   - [React Form Renderer](#react-form-renderer)
-  - [PatternFly 3 Mapper](#patternfly-3-mapper)
```

Comment:
What about to add 'mappers' section here?

Reply:
I don't think we need it here. Its mentioned in the root Readme and each mapper has its own section. This should be purely used for the renderer.

Reply:
The readme is mirrored into NPM, so when somebody comes from it, it would be nice to show him that there are several mapper he can use.

Reply:
Shoot you are right. It's this one.